src/test/java/Pages/AddUserPage.java

```diff
@@ -0,0 +1,36 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+
+public class AddUserPage {
+  WebDriver driver;
+
+  public AddUserPage(WebDriver driver){
```

Comment:
I suggest adding BasePage class, store there WebDriver as protected and create constructor that will set it up. Then you pages can extend it so you don't need to have WebDriver defined in every single page.